reaper_test.go

```diff
@@ -131,59 +117,55 @@ func TestContainerStartsWithoutTheReaper(t *testing.T) {
 	})
 	CleanupContainer(t, ctr)
 	require.NoError(t, err)
+}
 
-	sessionID := core.SessionID()
+// testReaperRunning validates that a reaper is running.
+func testReaperRunning(t *testing.T) {
+	t.Helper()
 
-	reaperContainer, err := lookUpReaperContainer(ctx, sessionID)
-	if err != nil {
-		t.Fatal(err, "expected reaper container not found.")
-	}
-	if reaperContainer != nil {
-		t.Fatal("expected zero reaper running.")
-	}
+	ctx := context.Background()
+	sessionID := core.SessionID()
+	reaperContainer, err := spawner.lookupContainer(ctx, sessionID)
+	require.NoError(t, err)
+	require.NotNil(t, reaperContainer)
 }
 
-func TestContainerStartsWithTheReaper(t *testing.T) {
-	config.Reset() // reset the config using the internal method to avoid the sync.Once
```

Comment:
I'm struggling to understand this, possible typo, lets discuss 😄

Reply:
No worries, I mentioned this as a reminder that, in general, I added config.Reset to many places as a means to write tests more easily, while in reality it causes more trouble than it solves. I wanted to provide context on that. That's why I marked it as offtopic, because I'll tackle that refactor of the config tests later on.